homeassistant/components/yale_smart_alarm/alarm_control_panel.py

```diff
@@ -99,6 +100,10 @@ def __init__(self, coordinator: YaleDataUpdateCoordinator) -> None:
             manufacturer=MANUFACTURER,
             model=MODEL,
             name=self._attr_name,
+            connections={
+                (CONNECTION_NETWORK_MAC, coordinator.data["panel_info"]["mac"])
+            },
+            sw_version=coordinator.data["panel_info"]["version"],
```

Comment:
Not that it saves you many lookups here but it would still be a bit cleaner to do a `panel_info = coordinator.data["panel_info"]` ....